docs/learn/learn-nft-pallets.md

```diff
@@ -12,61 +12,127 @@ NFT-related actions within their dApp.
 
 ## NFTs Pallet
 
-### Current Implementations
-
 NFTs is a [FRAME pallet](https://polkadot.js.org/docs/substrate/extrinsics#nfts) currently deployed
 on Westmint, which provides a multitude of functions to interact with NFTs.
 
 The pallet comes with a new way to configure NFTs, as well as configure collections and items.
 Pallet-level [feature flags](https://github.com/paritytech/substrate/pull/12367) allow disabling
 functionalities that are not needed in the runtime.
 
-Some features are common to the NFT world, such as:
+### Creating a Collection
+
+You can use the NFTs pallet to create NFTs collections. In the Polkadot-JS UI, go to Developer >
+Extrinsic and select the `nfts.create` extrinsic. When you create a collection, you must specify who
+the admin is. Then, under `config: PalletNftsCollectionConfig`, you can configure your collection's
+specifying different settings:
+
+- `settings` you can specify (in a bitflag-format) settings for your collection:
```

Comment:
I see that I can only enter a positive number `settings: u64` here. Let's say I would like to disable `Transferrable items` - what would be the numeric value for `settings: u64` ? As none of the settings below are visible on the UI, an example would make sense (on how to set the numeric value for settings).

Reply:
Polkadot JS does not support bitflags. Settings are possible but at the moment only the default "0" is avaialble.

Reply:
Got it! Let us add this as an info note here.